packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -393,7 +393,7 @@ export default defineComponent({
         client: this.$client,
         path: this.file.path,
         $gettext: this.$gettext,
-        ...(this.currentStorageId && { storageId: this.currentStorageId })
+        storageId: this.file.fileId
```

Comment:
Please remove the `currentStorageId` export from the setup function (line 183) since it's not used outside of the setup function anymore with this change.